components/listingsPage/filters.js

```diff
@@ -223,7 +223,11 @@ const Filters = ({ t }) => {
             <TooltipText
               minWidth="200px"
               side="left"
-              text={t("toolTips.age")}
+              text={
+                <Paragraph color="#515151" textSize="14px">
+                  {t("toolTips.age")}
+                </Paragraph>
+              }
```

Comment:
Can you remove the margin for each of the `Paragraphs` used inside the Tooltips? The padding on the tooltip doesn't look uniform like this. Maybe worth creating a custom `styled(Paragraph)` element and also apply the text size there and reuse it?